srcMain/main.cpp

```diff
@@ -488,13 +489,10 @@ void shasta::main::assemble(
     cout << timestamp << "Done loading reads from " << inputFileNames.size() << " files." << endl;
     cout << "Read loading took " << seconds(t1-t0) << "s." << endl;
 
-
-    // Create a histogram of read lengths.
+    // Log the read length histogram.
```

Comment:
I thought "Log" means "write to stdout". So this comment seems a bit misleading.

Reply:
Does it? In my mind, it means, log it to the logging system, which can be anything really. I can clarify as it is confusing to you.

Reply:
True, but the only logging system we have is writing to `stdout`. Although you could consider our various `csv` output files a form of logging. Minor point anyway - your choice.